vignettes/EpiNow2.Rmd

```diff
@@ -293,3 +293,69 @@ estimates$summary$summary_plot
 ```
 
 ![plot of chunk plot_regional_epinow_summary](EpiNow2-plot_regional_epinow_summary-1.png)
+
+If instead, the regions have different settings, the `opts_list()` function can be used. Here is an example where we assign testland a different ascertainment rate of 0.5, which by default is 1.
+
+
+``` r
+obs <- opts_list(
+  obs_opts(),
+  reported_cases,
+  testland = obs_opts(scale = Fixed(0.5))
+)
+```
+
+
+``` r
+estimates <- regional_epinow(
```

Comment:
my point from the last review was to merge this to only have a single instance of using regional_epinow. i.e just update ascertainment in the example above

Reply:
See a9e8ef9a9987683c729770374e2dea17b445c937 if it's what you mean.